src/components/layout/Footer.tsx

```diff
@@ -23,28 +25,31 @@ const Footer: React.FC<Props> = ({ children }) => {
                                         layout="fixed"
                                         width={175}
                                         height={101}
-                                        src="/thriveLogoFooter.png"
+                                        src={`https:${settings.logo.fields.file.url}`}
                                         alt="Thrive Academy"
                                     />
                                 </a>
                             </Link>
                             <ul className="list-unstyled">
                                 <li className="mb-2">West Des Moines, IA</li>
-                                <li className="mb-2">
-                                    <Link href="https://www.facebook.com/groups/1138474949915990/">
-                                        <a>
-                                            <FaFacebook /> Thrive on Facebook
-                                        </a>
-                                    </Link>
-
-                                </li>
-                                <li className="mb-2">
-                                    <Link href="mailto:thriveacademyia@gmail.com">
-                                        <a>
-                                            <FaEnvelope /> thriveacademyia@gmail.com
-                                        </a>
-                                    </Link>
-                                </li>
+                                {settings.facebook !== undefined && settings.facebook.length > 0 &&
```

Comment:
Super minor, but we could probably shorten these undefined/empty checks by checking the truthiness of the property
```suggestion
                                {settings.facebook &&
```
This should catch `undefined` values and empty strings. You could also use `settings.facebook?.trim()` to ensure empty strings aren't rendered if you'd like.

This is looking awesome!! Really appreciate all the work you're doing.